`make report-*` fails because `pk` sources not cloned

See here:

* https://github.com/riscv-collab/riscv-gnu-toolchain/issues/1087#issuecomment-1268663890

I recently had cause to try to run the test suite using `make report-newlib` and found that the issue remains extant.
For some reason the `riscv-gnu-toolchain` `Makefile` fails to retrieve/clone the `pk` sources so the build fails until the `pk` sources are manually cloned.

I'll post back with more info when I have any.

Having done that I get the following:
```
make[1]: Leaving directory '/home/tommy/Downloads/riscv-gnu-toolchain/build-spike'
mkdir -p stamps/
date > stamps/build-spike
rm -rf stamps/build-pk64 build-pk64
mkdir build-pk64
cd build-pk64 && /home/tommy/Downloads/riscv-gnu-toolchain/pk/configure \
	--prefix=/home/tommy/Downloads/riscv-gnu-toolchain/installed-tools \
	--host=riscv64-unknown-elf \
	--with-arch=rv64gc
/bin/sh: 1: /home/tommy/Downloads/riscv-gnu-toolchain/pk/configure: not found
make: *** [Makefile:866: stamps/build-pk64] Error 127
```
Obviously it changes to the `pk` directory and tries to `configure` but the directory is empty because the sources were never previously cloned.

Logs:

* [make-newlib.log.tar.gz](https://github.com/riscv-collab/riscv-gnu-toolchain/files/11565245/make-newlib.log.tar.gz)
* [make-report-newlib.log.tar.gz](https://github.com/riscv-collab/riscv-gnu-toolchain/files/11565246/make-report-newlib.log.tar.gz)

--- a/Makefile.in
+++ b/Makefile.in
@@ -270,6 +270,7 @@ endif
 
 ifeq ($(findstring $(srcdir),$(SPIKE_SRCDIR)),$(srcdir))
 SPIKE_SRC_GIT := $(SPIKE_SRCDIR)/.git
+PK_SRC_GIT := $(PK_SRCDIR)/.git
 else
 SPIKE_SRC_GIT :=
 endif
@@ -847,7 +848,7 @@ stamps/build-spike: $(SPIKE_SRCDIR) $(SPIKE_SRC_GIT)
 	mkdir -p $(dir $@)
 	date > $@
 
-stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
+stamps/build-pk32: $(PK_SRCDIR) $(PK_SRC_GIT) stamps/build-gcc-newlib-stage2
 	rm -rf $@ $(notdir $@)
 	mkdir $(notdir $@)
 	cd $(notdir $@) && $</configure \
@@ -860,7 +861,7 @@ stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
 	mkdir -p $(dir $@)
 	date > $@
 
-stamps/build-pk64: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
+stamps/build-pk64: $(PK_SRCDIR) $(PK_SRC_GIT) stamps/build-gcc-newlib-stage2
 	rm -rf $@ $(notdir $@)
 	mkdir $(notdir $@)
 	cd $(notdir $@) && $</configure \

